pkg/apis/batch/validation/validation_test.go

```diff
@@ -828,7 +854,7 @@ func TestValidateJob(t *testing.T) {
 
 	for k, v := range errorCases {
 		t.Run(k, func(t *testing.T) {
-			errs := ValidateJob(&v, JobValidationOptions{})
+			errs := ValidateJob(&v, JobValidationOptions{AllowBatchLabel: true})
```

Comment:
the options should be part of the testcase

Reply:
I made this change but it was a pretty large one.  I think its good long term.